packages/ui-kit/src/components/Switch/Switch.d.ts

```diff
@@ -0,0 +1,21 @@
+import { FC } from 'react';
+
+export interface SwitchProps {
+  /** label rigth switch */
+  label1?: string;
+  /** label left switch */
+  label2?: string;
+  /** Switch size */
+  size?: 'small' | 'medium' | 'large';
+  /** color switch*/
+  color?: 'primary' | 'secondary';
+  /**function used when switch is on */
+  on?: (e: object) => void;
```

Comment:
chamar funções handler (on/off), só de onChange.
quando for true, é on, quando for false é off.